Evgeniya_Bogdanova/js/scripts.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * Created by John Doe on 17.10.2017.
+ */
+document.onreadystatechange = function () {
+  if (document.readyState === 'interactive') {
+    initCalculator();
+  }
+};
+
+function initCalculator() {
+  var
+    calculator = document.getElementById('calculator'),
+    first = document.getElementById('first-number'),
+    second = document.getElementById('second-number'),
+    action = document.getElementById('action');
+
+  calculator.onsubmit = function(event) {
+    var
+      firstNumber = parseFloat(first.value),
+      secondNumber = parseFloat(second.value),
+      actionType = action.value.trim(),
+      result = calculate(firstNumber, secondNumber, actionType);
+
+    event.preventDefault();
+
+    resultRender(firstNumber, secondNumber, actionType, result);
+    this.reset();
+  }
+}
+
+function calculate(firstNumber, secondNumber, action) {
+  var result; 
+    
+  	switch(action)
+  	{
+	case '+':
+	case 'add':
+	  result = firstNumber + secondNumber;
+	  break;
+	case '-':
+	case 'minus':
+	  result = firstNumber - secondNumber;
+	  break;
+	case '*':
+	case 'multiply':
+	  result = firstNumber * secondNumber;
+	  break;
+	case '/':
+	case 'divide':
+	  result = firstNumber / secondNumber;
+	  break;
+	case '^':
+	case 'elevate':
+	  result = firstNumber ^ secondNumber;
```

Comment:
Operator `^` is binary `XOR`, like `&` (binary `AND`) and `|` (binary 'OR').
But don't confuse it with **logical** `&&` and `||`.
Please, use `Math.pow` function instead:
`a ^ b` --> `Math.pow(a, b)`;